packages/genesys-spark-components/src/components/stable/gux-content-search/gux-content-search.tsx

```diff
@@ -203,7 +203,7 @@ export class GuxContentSearch {
   }
 
   private showNavigationPanel(): boolean {
-    return this.value !== '' ? true : false;
+    return this.value !== '';
```

Comment:
I know I should not be making code changes in these Playwright PRs but this ternary boolean return is a pet peeve of mine :).